### ISSUE TITLE
LC Review

### ISSUE THREAD

**Issue by RuslanLikhobaba** [WUKS87/text-highlight#1]
![image](https://user-images.githubusercontent.com/86246718/233643106-689d4975-6cec-43b8-9459-d9458b5552bc.png)

**Comment by RuslanLikhobaba** [comment]
using ViewEncapsulation.None is a bad practice
[https://alex-klaus.com/angular-code-review/](Don’t use ViewEncapsulation.None (see [docs](https://angular.io/guide/component-styles#view-encapsulation)), as it’s the worst option (very prone to CSS leakage, which is hard to identify).)

**Comment by WUKS87** [comment]
> using ViewEncapsulation.None is bad practice [https://alex-klaus.com/angular-code-review/](Don’t use ViewEncapsulation.None (see [docs](https://angular.io/guide/component-styles#view-encapsulation)), as it’s the worst option (very prone to CSS leakage, which is hard to identify).)

I agree, just I thought to create minimal functionality per requirement. I can easily adjust.

@RuslanLikhobaba Removed Encapsulation config. Thanks

**Comment by WUKS87** [comment]
> ![image](https://user-images.githubusercontent.com/86246718/233643106-689d4975-6cec-43b8-9459-d9458b5552bc.png)

Ok, I didn't cover that case properly, I will have to work on it to cover it all.
